Dockerfile.cypress

```diff
@@ -0,0 +1,20 @@
+# base image
+FROM cypress/browsers:chrome65-ff57
+
+# set working directory
+RUN mkdir /usr/src/app
```

Comment:
Docker already has the /usr/src/app directory created so we can do WORKDIR only.